lib/backburner.js

```diff
@@ -30,6 +30,16 @@ export default function Backburner(queueNames, options) {
     end: [],
     begin: []
   };
+
+  var backburner = this;
+  this.executeTimers = function () {
```

Comment:
Because the point is to bind to this so we don't remake the bound function every time.